app/test_main.py

```diff
@@ -1,3 +1,57 @@
+import pytest
+from typing import Any
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected", [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17])
+    ],
+    ids=[
```

Comment:
it working but I'm not sure that it's a good idea to write IDs in different lists. As for me, this solution will be better when you write IDs in the next string after your arguments for the tests

Reply:
agree with Viktoriia, please fix it :)

Reply:
when ids are put together with args it's more clear what and how you want to test